src/os/ui/slick/slickgrid.js

```diff
@@ -452,6 +437,53 @@ os.ui.slick.SlickGridCtrl.prototype.disposeInternal = function() {
 };
 
 
+/**
+ * @param  {boolean} newValue
+ * @param  {boolean=} oldValue
+ * @private
+ */
+os.ui.slick.SlickGridCtrl.prototype.onDragEnabledChange_ = function(newValue, oldValue) {
+  if (newValue != oldValue) {
+    if (newValue) {
+      this.grid.onDragInit.subscribe(this.onDragInitFn_);
+      this.grid.onDragStart.subscribe(this.onDragStartFn_);
+      this.grid.onDrag.subscribe(this.onDragFn_);
+      this.grid.onDragEnd.subscribe(this.onDragEndFn_);
+
+      /**
+       * @type {goog.Timer}
+       * @protected
+       */
+      this.scrollTimer = new goog.Timer(20);
```

Comment:
Declaring new member properties here is weird now, but won't be allowed once we go to ES6 because it's flat out disallowed to do this outside of the constructor.

Reply:
good to know about that in ES6